packages/jest-matchers/src/__tests__/assertion-counts-test.js

```diff
@@ -35,4 +35,10 @@ describe('.hasAssertions()', () => {
     jestExpect.hasAssertions();
     jestExpect('a').toBe('a');
   });
+
+  it('throws if passed parameters', () => {
+    jestExpect(() => {
+      jestExpect.hasAssertions(2);
+    }).toThrow();
```

Comment:
Can you use `.toThrowError(/an error message/)`? It would be nice to know what do we throw 🙂